hip/GridInit.cpp

```diff
@@ -134,6 +153,10 @@ SimulationData grid_init_do_not_profile( Inputs in, int mype )
 	}
 	*/
 	
+    // Allocate Verification Array
+    size_t sz = in.lookups * sizeof(unsigned long);
+    SD.verification = (unsigned long *) malloc(sz);
+    nbytes += sz;
```

Comment:
Looks like there are some issues with indentation here. XSBench generally uses 8-wide tabs for indenting. New lines throughout this PR are indented with four spaces.